docs/code-and-design-guide-style-and-conventions/design-tokens.md

```
@@ -0,0 +1,42 @@
+# Design Tokens for Kindfi
+
+This document provides a description of the design tokens exported from Figma.
+
+## Colors
+
+### Light Mode
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)